### PR TITLE
Reduce time to interactive

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/components/jsx-helpers/jsx-helpers.jsx [hunk 1]
```diff
@@ -66,12 +66,37 @@ export function LoaderPage({
...
+        newScript.async = false;
+        s.parentNode?.replaceChild(newScript, s);
+
+        p.then(processOne);
+    });
+
+    processOne();
+};
+
 export function RawHTML({Tag='div', html, embed=false, ...otherProps}) {
```
Comment (nathanstitt) [inline src/app/components/jsx-helpers/jsx-helpers.jsx:94]: I've used https://www.npmjs.com/package/loadjs for things like this before

File: src/app/components/shell/welcome/welcome-content.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    function skipWalkthrough() {
+        cookie.setKey('walkthroughDone');
+        goToMyOpenStax();
+    }
+
+    function showWalkthrough() {
+        window.localStorage.showMyOpenStaxWalkthrough = true;
+        goToMyOpenStax();
+    }
```
Comment (nathanstitt) [inline src/app/components/shell/welcome/welcome-content.js:30]: Keep in mind that safari won't allow localStorage in an incognito window, so don't do anything that depends on it.  I've also been using https://github.com/astoilkov/use-local-storage-state which is pretty nice

**Review by nathanstitt — APPROVED** [review]

File: src/app/components/jsx-helpers/jsx-helpers.jsx (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/components/jsx-helpers/jsx-helpers.jsx:94]: I think that's for a different purpose. This is to allow RawHTML with `<script>` pieces -- usually Wistia video players.

Comment (nathanstitt) [reply src/app/components/jsx-helpers/jsx-helpers.jsx:94]: hah, I thought you were inserting script tags to load js

**Pull request merged**